ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -165,43 +175,61 @@ public void setUp() {
   }
 
   @Test
-  public void testFormatSql() {
-
-    final ArrayList<TableElement> tableElements = new ArrayList<>();
-    tableElements.add(new TableElement("GROUP", PrimitiveType.of(SqlType.STRING)));
-    tableElements.add(new TableElement("NOLIT", PrimitiveType.of(SqlType.STRING)));
-    tableElements.add(new TableElement("Having", PrimitiveType.of(SqlType.STRING)));
-
+  public void shouldFormatCreateStreamStatementWithImplicitKey() {
+    // Given:
     final CreateStream createStream = new CreateStream(
         QualifiedName.of("TEST"),
-        tableElements,
+        ELEMENTS_WITHOUT_KEY,
         false,
-        ImmutableMap.of(
-            DdlConfig.KAFKA_TOPIC_NAME_PROPERTY, new StringLiteral("topic_test"),
-            DdlConfig.VALUE_FORMAT_PROPERTY, new StringLiteral("avro")
-        ));
+        SOME_WITH_PROPS);
+
+    // When:
     final String sql = SqlFormatter.formatSql(createStream);
-    assertThat("literal escaping failure", sql, containsString("`GROUP` STRING"));
-    assertThat("not literal escaping failure", sql, containsString("NOLIT STRING"));
-    assertThat("lowercase literal escaping failure", sql, containsString("`Having` STRING"));
-    final List<PreparedStatement<?>> statements = KsqlParserTestUtil.buildAst(sql,
-        MetaStoreFixture.getNewMetaStore(mock(FunctionRegistry.class)));
-    assertFalse("formatted sql parsing error", statements.isEmpty());
+
+    // Then:
+    assertThat(sql, is("CREATE STREAM TEST (Foo STRING, Bar STRING) "
+        + "WITH (VALUE_FORMAT='JSON', KAFKA_TOPIC='topic_test', KEY='ORDERID');"));
```

Comment:
Remove the `KEY` since it's not in the table elements!

Reply:
This is what the formatter outputs. So I can't remove KEY as the test would fail.